datafusion/expr-common/src/interval_arithmetic.rs

```diff
@@ -76,6 +76,14 @@ macro_rules! get_extreme_value {
             DataType::Interval(IntervalUnit::MonthDayNano) => {
                 ScalarValue::IntervalMonthDayNano(Some(IntervalMonthDayNano::$extreme))
             }
+            DataType::Decimal128(precision, scale) => {
+                ScalarValue::Decimal128(Some(i128::$extreme), *precision, *scale)
```

Comment:
Is this correct for min/max of i128? It seems like the minimum value of Decimal128(precision, scale) would be the minimum value for the precision and scale separately rather than the min value of the overall i128 🤔

Reply:
I dug around in arrow a bit and I think these are the min/max values for each decimal precision

https://docs.rs/arrow/latest/arrow/datatypes/constant.MIN_DECIMAL_FOR_EACH_PRECISION.html
https://docs.rs/arrow/latest/arrow/datatypes/constant.MAX_DECIMAL_FOR_EACH_PRECISION.html

Reply:
Ahh, that makes sense. These huge arrays remind me it should be all of `9`s...

Reply:
sadly the decimal 256 version is not public 😢 let me find a workaround

Reply:
maybe `get_extreme_value` should be part of arrow?

Reply:
> sadly the decimal 256 version is not public 😢 let me find a workaround

I recommend 
1.  copy/pasting the value into DataFusion for now
2. File a ticket in arrow-rs to make it public (or maybe even a PR!)
3. leave a comment next to the copy in DataFUsion referencing the upstream ticket

Reply:
I filed a PR in arrow to expose these constants:
-  https://github.com/apache/arrow-rs/pull/6992